compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -5432,9 +5432,12 @@ private BLangMarkdownDocumentation createMarkdownDocumentationAttachment(Optiona
 
                     BLangIdentifier paraName = new BLangIdentifier();
                     Token parameterName = parameterDocLineNode.parameterName();
-                    paraName.value = parameterName.isMissing() ? "" : parameterName.text();
+                    var parameterNameValue = parameterName.isMissing() ? "" : parameterName.text();
                     bLangParaDoc.parameterName = paraName;
-
+                    if (stringStartsWithSingleQuote(parameterNameValue)) {
+                        parameterNameValue = parameterNameValue.substring(1);
+                    }
+                    paraName.value = parameterNameValue;
```

Comment:
Adding these lines before `bLangParaDoc.parameterName = paraName;` is more cleaner right?